pages/settings/_submenu.vue

```diff
@@ -39,8 +40,4 @@
 
 <style scoped lang="scss">
 .component { background: map-get($black, "lighten") !important; }
-
-// <overlay-scrollbars/> loses all its padding when destroyed, causing a visual jump
-// The solution: use margins. Unfortunately, one of its inner elements has a top padding, which we disable here.
-*::v-deep .os-content { padding-top: 0 !important; }
```

Comment:
Speaking of which, the spacings between content (and screen/layout edges) are all the same?

Reply:
I didn't measure anything but nothing looks too high or low. Even when the window is really small the scrollbar reaches the edges of the container as expected.

Reply:
What about on mobile?